src/components/WidgetStats/WidgetStatsData.js

```diff
@@ -1,70 +1,9 @@
-import { useQuery } from '@tanstack/react-query'
-
 // FPCC
-import { useSiteStore } from 'context/SiteContext'
-import api from 'services/api'
+import { useStats } from '../../common/dataHooks/useStats'
```

Comment:
Avoid using relative imports. You can leverage our Webpack aliases here and change this to `'common/dataHooks/useStats'`